src/EnergyPlus/DisplayRoutines.cc

```diff
@@ -93,6 +93,8 @@ void DisplayString(std::string const &String) // String to be displayed
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     // na
+    if (DataGlobals::fMessagePtr) DataGlobals::fMessagePtr(String);
+    if (DataGlobals::messageCallback) DataGlobals::messageCallback(String.c_str());
```

Comment:
`DisplayString()` was not using the callbacks, this just reports those strings out to the callbacks when needed.